<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Main</class>
 <widget class="QDialog" name="Main">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>950</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
padding: 0;
margin: 0;
background-color: rgb(32, 31, 31);
color: #fff;
}

#Header{
background-color: rgb(35, 37, 43);
}

#Dialog{
background-color: rgb(35, 37, 43);
border-radius: 15px;
}

#title {
color: rgba(100, 189, 100, 1);
}

#InputDialog{
border-radius: 10px;
background-color:#282C34;
}

#line_2, #line_3, #line_4, #line_5{
background-color: #494949;
}

#close, #collapse{
background-color: #23252B;
border-style:solid;
}

#settingBtn, #logoutBtn{
border-style:solid;

}

#optionsTxt, #advSetTxt, #tempTxt, #tempResTxt {
background-color:#282C34;
}

#clearBtn, #generationBtn {
background-color: #28F1A9;
border-style: solid;
border-radius: 10px;
color: #000;
}

#droparrowBtn{
border-style: solid;
background-color: #23252B;
}

#droparrowBtn:pressed{
background-color: #23252B;
}

#headlineEdit, #audienceEdit, #keyEdit, #actionEdit, #styleBox, #lengthEdit{
border-style:solid;
border-width:1px;
border-color:#494949;
}

</string>
  </property>
  <widget class="QWidget" name="Header" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>950</width>
     <height>50</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QPushButton" name="collapse">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>15</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../images/collapse.png</normaloff>../images/collapse.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="close">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>15</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../images/close.png</normaloff>../images/close.png</iconset>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="Body" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>50</y>
     <width>950</width>
     <height>800</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QLabel" name="icon">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>80</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../images/icon.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="title">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>AdForgeAI</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>290</y>
      <width>171</width>
      <height>221</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="settingTxt">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>100</y>
       <width>90</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Настройки</string>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="settingBtn">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <kerning>true</kerning>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../images/setting.png</normaloff>../images/setting.png</iconset>
     </property>
     <property name="autoDefault">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>155</width>
       <height>4</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="lineWidth">
      <number>2</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="logoutTxt">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>180</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Выйти</string>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="logoutBtn">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>180</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <kerning>true</kerning>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../images/logout.png</normaloff>../images/logout.png</iconset>
     </property>
     <property name="autoDefault">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <zorder>frame</zorder>
   <zorder>icon</zorder>
   <zorder>title</zorder>
  </widget>
  <widget class="QWidget" name="Dialog" native="true">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>70</y>
     <width>730</width>
     <height>700</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QPushButton" name="droparrowBtn">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>150</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../images/up-arrow.png</normaloff>../images/up-arrow.png</iconset>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QFrame" name="line_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>150</y>
      <width>500</width>
      <height>2</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QFrame" name="line_2">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>0</y>
      <width>2</width>
      <height>700</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="InputDialog" native="true">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>240</y>
     <width>480</width>
     <height>510</height>
    </rect>
   </property>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>481</width>
      <height>2</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="optionsTxt">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>55</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Основные параметры</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="headlineEdit">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>90</y>
      <width>415</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Введите заголовок объявления</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="audienceEdit">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>135</y>
      <width>415</width>
      <height>26</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Введите целевую аудитория</string>
    </property>
   </widget>
   <widget class="QPushButton" name="generationBtn">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>6</y>
      <width>165</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Генерировать объявление</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clearBtn">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>6</y>
      <width>90</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">w</string>
    </property>
    <property name="text">
     <string>Очистить всё</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="keyEdit">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>180</y>
      <width>415</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Введите ключевые преимущества</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="actionEdit">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>225</y>
      <width>415</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Введите призыв к действию</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lengthEdit">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>315</y>
      <width>415</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Введите ограничение на длину слов</string>
    </property>
   </widget>
   <widget class="QFrame" name="line_5">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>380</y>
      <width>350</width>
      <height>2</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QLabel" name="advSetTxt">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>400</y>
      <width>170</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Расширенные настройки</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="styleBox">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>270</y>
      <width>415</width>
      <height>26</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Выберите стиль объявления</string>
    </property>
    <property name="whatsThis">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">QToolTip{
color: black;
background-color:white;
font-size:12px;
}</string>
    </property>
    <property name="editable">
     <bool>false</bool>
    </property>
    <property name="duplicatesEnabled">
     <bool>false</bool>
    </property>
    <item>
     <property name="text">
      <string>Деловой</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Неформальный</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Юмористический</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="styleBox_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>430</y>
      <width>141</width>
      <height>26</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Выберите модель ИИ</string>
    </property>
    <property name="whatsThis">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">QToolTip{
color: black;
background-color:white;
font-size:12px;
}</string>
    </property>
    <property name="editable">
     <bool>false</bool>
    </property>
    <property name="duplicatesEnabled">
     <bool>false</bool>
    </property>
    <item>
     <property name="text">
      <string>gpt-3.5-turbo</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>gpt-4-turbo</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>gpt-4</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>google/gemma-1.1-7b-it</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>gemini-1-5-pro</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>gemini-pro</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="styleBox_3">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>430</y>
      <width>151</width>
      <height>26</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Потоковая обработка</string>
    </property>
    <property name="whatsThis">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">QToolTip{
color: black;
background-color:white;
font-size:12px;
}</string>
    </property>
    <property name="editable">
     <bool>false</bool>
    </property>
    <property name="duplicatesEnabled">
     <bool>false</bool>
    </property>
    <item>
     <property name="text">
      <string>Выключить обработку</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Включить обработку</string>
     </property>
    </item>
   </widget>
   <widget class="QSlider" name="slider">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>470</y>
      <width>201</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QSlider{
background-color: #282C34;
}

QSlider::groove:horizontal {
                height: 1px;
                background: #53F4BA;
                margin: 2px 0;
                border-radius: 2px;
            }

            QSlider::handle:horizontal {
                background: #53F4BA;
                border: 1px solid #53F4BA;
                width: 7px;
                height: 7px;
                border-radius: 3px;
                margin: -5px 0;
            }

            QSlider::sub-page:horizontal {
                background: #53F4BA;
                border: 1px solid #53F4BA;
                height: 8px;
                border-radius: 4px;
            }

            QSlider::add-page:horizontal {
                background: #222429;
                border: 1px solid #222429;
                height: 8px;
                border-radius: 4px;
}</string>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="tempTxt">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>472</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Температура</string>
    </property>
   </widget>
   <widget class="QLabel" name="tempResTxt">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>474</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0.0</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
